Sources/TuistGraph/Models/CopyFilesAction.swift

```diff
@@ -13,6 +13,8 @@ public struct CopyFilesAction: Equatable, Codable {
 
     /// Relative paths to the files to be copied.
     public var files: [FileElement]
+    
+    public var platformFilters: [PlatformFilter]?
```

Comment:
Can you please also add swift doc as for the other fields?
```suggestion
    public var platformFilters: [PlatformFilter]
```

Reply:
As this is a very early stage prototype, i'll add comment docs intended to facilitate review but i expect the need to iterate a bit so would prefer investment here to wait until later.